test/functional/mempool_persist.py

```diff
@@ -159,6 +159,15 @@ def run_test(self):
         assert self.nodes[0].getmempoolinfo()["loaded"]
         assert_equal(len(self.nodes[0].getrawmempool()), 0)
 
+        self.log.debug("Import mempool at runtime to node0.")
+        assert_equal({}, self.nodes[0].importmempool(mempooldat0))
+        assert_equal(len(self.nodes[0].getrawmempool()), 7)
+        fees = self.nodes[0].getmempoolentry(txid=last_txid)["fees"]
+        assert_equal(fees["base"], fees["modified"])
+        assert_equal({}, self.nodes[0].importmempool(mempooldat0, {"apply_fee_delta_priority": True, "apply_unbroadcast_set": True}))
+        fees = self.nodes[0].getmempoolentry(txid=last_txid)["fees"]
```

Comment:
I'd like the test to load against two mempool files with differing txn subsets at least, showing that "additional" txs are loaded even if they weren't there originally. Even better if they're not strict subsets.

Reply:
Wrote a test, feel free to take / not take: https://github.com/glozow/bitcoin/commit/893f79c5ff612122e5b98d65b9859e7260c51a95#diff-99a8c43ebc6b8546e5fb9bf74032e96a6d3376b65655837e12454425098b9ad3R222